wgpu-core/src/device/queue.rs

```diff
@@ -56,7 +69,8 @@ crate::impl_storage_item!(Queue);
 impl<A: HalApi> Drop for Queue<A> {
     fn drop(&mut self) {
         resource_log!("Drop {}", self.error_ident());
-        let queue = self.raw.take().unwrap();
+        // Safety: we never access `self.raw` beyond this point.
```

Comment:
suggestion: We should use the same case as suggested by Rust upstream for `SAFETY` comments, i.e., uppercase.

```suggestion
        // SAFETY: we never access `self.raw` beyond this point.
```

Reply:
Pushed up <https://github.com/gfx-rs/wgpu/commits/825400c086fb029f93e08b8b4e0e005759a755d3> to fulfill the above.

Reply:
suggestion: The same page as ☝🏻 also shows an example of a `# Safety` section in API documentation, which doesn't include a trailing colon in the section name. We do this in some places. I've also pushed <https://github.com/gfx-rs/wgpu/commit/1787279aa3c16c0050dc148da2331586a1f0f865> to address this.

>